01-gabbar/00-tekton-pipelines/00-build/values.yaml

```diff
@@ -38,10 +38,22 @@ pipeline-charts:
       - taskName: stakater-sonarqube-scanner-v1
         runAfter:
           - stakater-unit-test-v1
```

Comment:
change this to stakater-build-image-flag

Reply:
i think the flow would be 

unit-test >> check-build-image >> ( build & push ) and ( sonarqube scanner )